### PR TITLE
try to fix broken logo and favicon

### PR DESCRIPTION
**Pull request by pjfanning**
see https://github.com/apache/incubator-pekko-site/issues/95

needs more testing before merging

### CODE REVIEW

**Comment by pjfanning** [comment]
@samueleresca @mdedetrich I built this locally and then used it in pekko-grpc paradox build (also locally) - but none of the assets.hostname dependent paths were mapped - all the links are relative. This includes the existing paths that already support assets.hostname.

**Comment by samueleresca** [comment]
> @samueleresca @mdedetrich I built this locally and then used it in pekko-grpc paradox build (also locally) - but none of the assets.hostname dependent paths were mapped - all the links are relative. This includes the existing paths that already support assets.hostname.
> 
> edit - I tested with pekko-site repo and these changes seem to work there

Have you tried to also specify the `assets.hostname` property into the downstream module (e.g.: [paradoxProperties in incubator-pekko-(http|grpc)](https://github.com/apache/incubator-pekko-http/blob/0dfd4ff9cee64d3dcf4b26164a69fa6363772e50/build.sbt#L434))

**Comment by pjfanning** [comment]
The `$page.properties.("assets.hostname")$$page.base$` pattern is already used in https://github.com/apache/incubator-pekko-sbt-paradox/blob/d47e88d12a922d678048306d9b069106162cb00f/theme/src/main/assets/page.st#L57

**Comment by mdedetrich** [comment]
> I can take a look at this tomorrow

Ill wait till tomorrow to see if there is an update on this

File: theme/src/main/assets/page.st [hunk 1]
```diff
@@ -46,7 +46,7 @@ $!
     $ elseif (page.properties.("material.author")) $
       <meta name="author" content="$page.properties.("site.author")$">
     $ endif $
-    <link rel="shortcut icon" href="$page.base$$page.properties.("material.favicon")$">
+    <link rel="shortcut icon" href="$page.properties.("assets.hostname")$$page.base$$page.properties.("material.favicon")$">
```
Comment (Roiocam) [inline theme/src/main/assets/page.st:49]: I doubt that this is related to https://github.com/apache/incubator-pekko-sbt-paradox/pull/86. I'm not sure if page.properties ("assets.hostname") exists, but using it with $page.base$ may be a bad idea.

Comment (Roiocam) [reply theme/src/main/assets/page.st:49]: The icon link of all modules is broken, but the pekko homepage is not, which also proves this. `$page.base$` is the path of a specific project.

Comment (Roiocam) [reply theme/src/main/assets/page.st:49]: > The link is to https://pekko.apache.org/docs/pekko-http/current/assets/images/pekko_logo.png
>
> Should be https://pekko.apache.org/assets/images/pekko_logo.png

The previous comments should be no problem. `$page.base$` is not a problem. At present, it is always the root directory of the project, that is.

The documents generated by all these modules (http, connector, pekko) are "assets/xxxx", even if the string template adds `assets.home`, but due to pekko, conn Ector These modules do not have the default value of `assets.home`, so they still do not point the path to CDN.

So I think the PR needs to set a default value for assets.home, or explicitly add the assets.home configuration value for each project.

**Comment by pjfanning** [comment]
seems to be included in #127

**Comment by samueleresca** [comment]
Added an issue tracking the `page.base` paths in the static assets: https://github.com/apache/incubator-pekko-sbt-paradox/issues/128

**Pull request closed without merging**